docs/book/src/component-config-tutorial/testdata/project/config/default/kustomization.yaml

```diff
@@ -26,24 +26,24 @@ resources:
 # [PROMETHEUS] To enable prometheus monitor, uncomment all sections with 'PROMETHEUS'.
 #- ../prometheus
 
-patchesStrategicMerge:
+patches:
```

Comment:
you can check that those changes are not working
You can either test the solution locally
Just go in the samples under testdata and call the targets
See the errors in the ci check like

```   {
          s: "make deploy IMG=e2e-test/controller-manager:tuel failed with error: (exit status 2) /home/prow/go/src/sigs.k8s.io/kubebuilder/test/e2e/v4/e2e-tuel/bin/controller-gen rbac:roleName=manager-role crd webhook paths=\"./...\" output:crd:artifacts:config=config/crd/bases\ntest -s /home/prow/go/src/sigs.k8s.io/kubebuilder/test/e2e/v4/e2e-tuel/bin/kustomize || GOBIN=/home/prow/go/src/sigs.k8s.io/kubebuilder/test/e2e/v4/e2e-tuel/bin GO111MODULE=on go install sigs.k8s.io/kustomize/kustomize/v5@v5.1.1\ncd config/manager && /home/prow/go/src/sigs.k8s.io/kubebuilder/test/e2e/v4/e2e-tuel/bin/kustomize edit set image controller=e2e-test/controller-manager:tuel\n/home/prow/go/src/sigs.k8s.io/kubebuilder/test/e2e/v4/e2e-tuel/bin/kustomize build config/default | kubectl apply -f -\nError: trouble configuring builtin PatchTransformer with config: `\npath: webhookcainjection_patch.yaml\n`: unable to parse SM or JSON patch from [# This patch add annotation to admission webhook config and\n# CERTIFICATE_NAMESPACE and CERTIFICATE_NAME will be substituted by kustomize\napiVersion: admissionregistration.k8s.io/v1\nkind: MutatingWebhookConfiguration\nmetadata:\n  labels:\n    app.kubernetes.io/name: mutatingwebhookconfiguration\n    app.kubernetes.io/instance: mutating-webhook-configuration\n    app.kubernetes.io/component: webhook\n    app.kubernetes.io/created-by: e2e-tuel\n    app.kubernetes.io/part-of: e2e-tuel\n    app.kubernetes.io/managed-by: kustomize\n  name: mutating-webhook-configuration\n  annotations:\n    cert-manager.io/inject-ca-from: CERTIFICATE_NAMESPACE/CERTIFICATE_NAME\n---\napiVersion: admissionregistration.k8s.io/v1\nkind: ValidatingWebhookConfiguration\nmetadata:\n  labels:\n    app.kubernetes.io/name: validatingwebhookconfiguration\n    app.kubernetes.io/instance: validating-webhook-configuration\n    app.kubernetes.io/component: webhook\n    app.kubernetes.io/created-by: e2e-tuel\n    app.kubernetes.io/part-of: e2e-tuel\n    app.kubernetes.io/managed-by: kustomize\n  name: validating-webhook-configuration\n  annotations:\n    cert-manager.io/inject-ca-from: CERTIFICATE_NAMESPACE/CERTIFICATE_NAME\n]\nerror: no objects passed to apply\nmake: *** [Makefile:138: deploy] Error 1\n",
```